Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -2328,9 +2328,20 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
     /// - Parameters:
     ///   - name: The name of the asset.
     ///   - parent: The topic where the asset is referenced.
+    ///   - type: If not nil, check the asset's extension type
```

Comment:
I’d suggest describing what the parameter represents, then what it means for this function if it’s not nil:

>The type of the asset to resolve. If an asset with such type could not be found in the context, this function returns nil.

Reply:
Yes, that sounds right, thank you!